binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -60,18 +60,16 @@
  * @author Oliver Libutzki
  * @author Kai Kreuzer - stabilized code
  * @author Andre Fuechsel - implemented switch off when brightness == 0
+ * @author Thomas Höfer - added thing properties
  *
  */
 public class HueLightHandler extends BaseThingHandler implements LightStatusListener {
 
-	public final static Set<ThingTypeUID> SUPPORTED_THING_TYPES = Sets
-			.newHashSet(THING_TYPE_LCT001, THING_TYPE_LCT002,
-					THING_TYPE_LCT003, THING_TYPE_LLC001, THING_TYPE_LLC006,
-					THING_TYPE_LLC007, THING_TYPE_LLC010, THING_TYPE_LLC011,
-					THING_TYPE_LLC012, THING_TYPE_LLC013, THING_TYPE_LWL001,
-					THING_TYPE_LST001, THING_TYPE_LCT003, THING_TYPE_LWB004,
-					THING_TYPE_CLASSIC_A60_RGBW, THING_TYPE_SURFACE_LIGHT_TW,
-					THING_TYPE_ZLL_LIGHT);
+    public final static Set<ThingTypeUID> SUPPORTED_THING_TYPES = Sets.newHashSet(THING_TYPE_LCT001, THING_TYPE_LCT002,
+            THING_TYPE_LCT003, THING_TYPE_LLC001, THING_TYPE_LLC006, THING_TYPE_LLC007, THING_TYPE_LLC010,
+            THING_TYPE_LLC011, THING_TYPE_LLC012, THING_TYPE_LLC013, THING_TYPE_LWL001, THING_TYPE_LST001,
+            THING_TYPE_LCT003, THING_TYPE_LWB004, THING_TYPE_CLASSIC_A60_RGBW, THING_TYPE_SURFACE_LIGHT_TW,
+            THING_TYPE_ZLL_LIGHT);
```

Comment:
@dnobel 
So, the formatter seems no to be applied to everything.

@tomhoefer 
The formatter could be changed to add a line break after every element, but this would bloat the code (add a huge new lines, if after every argument a new line is started).
The formatter could be changed to indent all arguments after a line break (regardless why a new line is used) by column. So in that case indent after the beginning braces). Perhaps this could look better, but if the braces is at the right end of a line, this will also add a lot of new line because every argument is forced to be in one line again.